buildSrc/build.gradle.kts

```diff
@@ -22,8 +22,8 @@ sourceSets.main {
 }
 
 dependencies {
-    implementation(kotlin("gradle-plugin", libs.versions.kotlin.version.get()))
-    implementation(kotlin("serialization", libs.versions.kotlin.version.get()))
+    implementation(kotlin("gradle-plugin", "1.6.21"))
```

Comment:
Let's leave a comment or create an issue in YT to not forget updating it.

Reply:
Done: https://youtrack.jetbrains.com/issue/KTOR-4463/Update-Kotlin-to-170-in-buildSrc